### PR TITLE
[AUT-251] Decrypt SMIME into MIME using KVConnectDecryption

### PR DESCRIPTION
**Pull request by misolubarda**

### CODE REVIEW

File: Krypt/Source/KVConnectSMIME.swift [hunk 1]
```diff
@@ -0,0 +1,41 @@
+//
+//  KVConnect.swift
+//  CryptoSwift
+//
+//  Created by Miso Lubarda on 29.04.19.
+//
+
+import Foundation
+
+public struct KVConnectSMIME {
```
Comment (thehungrydev) [inline Krypt/Source/KVConnectSMIME.swift]:
```suggestion
public struct KVConnect {
```
I think `SMIME` is implementation detail, should be a blackbox for the consumer. Wyt?

File: Krypt/Source/KVConnectSMIME.swift [hunk 2]
```diff
@@ -0,0 +1,41 @@
+//
+//  KVConnect.swift
+//  CryptoSwift
```
Comment (thehungrydev) [inline Krypt/Source/KVConnectSMIME.swift]:
```suggestion
//  Krypt
```

File: Krypt/Source/CACertificates.swift [hunk 3]
```diff
@@ -0,0 +1,27 @@
+//
+//  CACertificates.swift
+//  CryptoSwift
```
Comment (thehungrydev) [inline Krypt/Source/CACertificates.swift]:
```suggestion
//  Krypt
```

File: Example/Tests/KVConnectSMIMETests.swift [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+//  Created by Miso Lubarda on 29.04.19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
+import Krypt
+import XCTest
+
+final class KVConnectSMIMETests: XCTestCase {
```
Comment (thehungrydev) [inline Example/Tests/KVConnectSMIMETests.swift]:
```suggestion
final class KVConnectTests: XCTestCase {
```

File: Example/Tests/KVConnectSMIMETests.swift [hunk 5]
```diff
@@ -0,0 +1,83 @@
+//
+//  KVConnectSMIMETests.swift
```
Comment (thehungrydev) [inline Example/Tests/KVConnectSMIMETests.swift]:
```suggestion
//  KVConnectTests.swift
```

File: Krypt/Source/KVConnectSMIME.swift (hunk 1 above)
Comment (misolubarda) [reply Krypt/Source/KVConnectSMIME.swift]: I think we should do it the other way arround: this is generic SMIME to MIME decryption and verification. It has nothing to do with KVConnect. We should rename everything to SMIMEDecryptor or something similar. What do you say?

**Review by markotect — APPROVED** [review]

**Pull request merged**